### PR TITLE
Fixed Issue #908

### PR DESCRIPTION
**Pull request by andresgaragiola**
Fixed issue #908.

### CODE REVIEW

**Comment by andresgaragiola** [comment]
DCO1.1 Signed-off-by: Andres Garagiola andresgaragiola@gmail.com

File: core.yaml [hunk 1]
```diff
@@ -389,7 +389,11 @@ security:
     # Can be 256 or 384. If you change here, you have to change also
     # the same property in membersrvc.yaml to the same value
     level: 256
-
+ 
+    # Can be SHA2 or SHA3. If you change here, you have to change also
+    # the same property in membersrvc.yaml to the same value
+    hashAlgorithm: SHA3
```
Comment (srderson) [inline core.yaml:395]: Any thoughts on whether https://github.com/hyperledger/fabric/blob/master/core/util/utils.go#L34 should read this same configuration setting or whether a new one should be created? @manish-sethi

**Comment by srderson** [comment]
LGTM. Confirmed all tests are passing. Thanks

File: core.yaml (hunk 1 above)
Comment (manish-sethi) [inline core.yaml:395]: @srderson, I think that it's better make the util method take these properties as parameters and then define properties separately for different usages (e.g., one we can define for state).

**Pull request merged**